### PR TITLE
Connect frontend backend

### PR DESCRIPTION
**Pull request by ethanratnofsky**
Refactored explore page to reflect new designs and use styles from Venue map card.

# Old

# New

# Features
* Search venues by name or address
* Refactor filters to sort by emoji
* Refactor trending venues to just be sorted by aggregated reaction count
* Conformed style of venue cards to that of venue map cards

### CODE REVIEW

File: src/components/explore/VenueCard.tsx
Comment (zineanteoh) [inline src/components/explore/VenueCard.tsx:1]: Is there a better name for this component? We already have `VenueCard` in `/components/map`. 

Maybe something like `VenueCardRow`?

File: src/components/explore/VenueReactButton.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    // Preemptively update the count to avoid a delay in the UI
+    setCount(prev => (didReact ? prev - 1 : prev + 1));
+    setDidReact(prev => !prev);
+
+    // TODO: Asynchronously update reaction in DB
+    console.log(
+      `TODO: Toggle reaction for ${reaction} on venue ${venue._id} for user _!`
+    );
+
+    // TODO: Revert the UI update if the DB update fails
```
Comment (zineanteoh) [inline src/components/explore/VenueReactButton.tsx:20]: If DB update fails we should also show the error message similar to what we have in signup/login page. 
(future TODO though)

**Review by zineanteoh — APPROVED** [review]
LGTM. Just few comments. 

Sorting is not expected to work yet, is it? Pressing on any emojis don't actually do the sorting.

**Review by zineanteoh — APPROVED** [review]
LGTM!

**Comment by ethanratnofsky** [comment]
> LGTM. Just few comments.
> 
> Sorting is not expected to work yet, is it? Pressing on any emojis don't actually do the sorting.

Will fix sorting in future when reactions are connected to database. Thinking about using a context for venues.

**Pull request merged**